azure-pipelines.yml

```
@@ -3,19 +3,25 @@ resources:
     - repository: templates
       type: github
       name: NSLS-II/profile-collection-ci
-      ref: refs/heads/main
+      ref: refs/heads/2021-3.1
```

https://github.com/NSLS-II/profile-collection-ci/pull/28 is merged to `main`, need to use the config from that branch.
```suggestion
      ref: refs/heads/main
```